src/plone/restapi/services/users/update.py

```diff
@@ -46,6 +46,14 @@ def _get_user(self, user_id):
         portal_membership = getToolByName(portal, 'portal_membership')
         return portal_membership.getMemberById(user_id)
 
+    def _change_user_password(self, user, value):
+        try:
+            user.userSetPassword(user.getUserId(), value)
+        except AttributeError:
+            # plone >= 5.2
+            acl_users = getToolByName(self.context, 'acl_users')
+            acl_users.userSetPassword(user.getUserId(), value)
```

Comment:
Any reason not to do it this way on older versions of Plone too?

Reply:
hmm, not really, I guess it works there also. 
Did not want to change previous behavior though. But would be cleaner. 
@tisto @lukasgraf opinions?

Reply:
If it works, I'm all for removing unnecessary conditionals.